### DISCUSSION TITLE
sqlite3.OperationalError: near "INTO": syntax error

### DISCUSSION THREAD

**Discussion by Mithra001** [apache/airflow#21176] [Q&A] [answered]
I'm trying to copy all database tables to new table. 
for that I tried this one : stm = c.execute("CREATE TABLE temp AS (SELECT * FROM main);") 
 Then I tried to create a new blank database and copy all tables from original to the new database. I tried the below one..
import sqlite3
import os
dirname = os.path.dirname(__file__)
database = os.path.join(dirname, "database/main.db")
con = sqlite3.connect(database)
c = con.cursor()
conn = sqlite3.connect('dbtemp.db')
c.execute("SELECT * INTO dbtemp.db.surface from main.db.surface;")
con.commit()
con.close()

Then I'm getting an error ' c.execute("SELECT * INTO dbtemp.db.surface from main.db.surface;")
sqlite3.OperationalError: near "INTO": syntax error'. 
Hope someone can help me to find where I made mistakes!!

**Comment by potiuk** [comment, marked as answer]
Maybe Google is there to help ?

The first result of google search of `sqlite select into` - says this: https://stackoverflow.com/questions/2361921/select-into-statement-in-sqlite
